packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -1595,15 +1595,15 @@ void AlgMJ<mj_scalar_t, mj_lno_t, mj_gno_t, mj_part_t, mj_node_t>::
     local_part_xadj(0) = static_cast<mj_lno_t>(num_selected_coords);
   });
 
-  auto local_coordinate_permutations =
-    this->coordinate_permutations;
-  auto local_inital_adjList_output_adjlist =
-    inital_adjList_output_adjlist;
-  Kokkos::parallel_for(
-    Kokkos::RangePolicy<typename mj_node_t::execution_space, mj_lno_t>
-      (0, num_total_coords), KOKKOS_LAMBDA (mj_lno_t i) {
-    local_coordinate_permutations(i) = local_inital_adjList_output_adjlist(i);
-  });
+  {
+    auto local_coordinate_permutations = this->coordinate_permutations;
+    auto local_initial_adjList_output_adjlist = initial_adjList_output_adjlist;
+    Kokkos::parallel_for(
+      Kokkos::RangePolicy<typename mj_node_t::execution_space, mj_lno_t>
+        (0, num_total_coords), KOKKOS_LAMBDA (mj_lno_t i) {
+      local_coordinate_permutations(i) = local_initial_adjList_output_adjlist(i);
```

Comment:
Why not just use `Kokkos::deep_copy`?  You're just copying a 1-D View into a 1-D View.

Reply:
Thanks for catching this oversight. Fixed.